crates/project_model/src/lib.rs

```diff
@@ -69,18 +69,17 @@ impl ProjectManifest {
     }
 
     pub fn discover_single(path: &AbsPath) -> Result<ProjectManifest> {
-        let mut candidates = ProjectManifest::discover(path)?;
-        let res = match candidates.pop() {
+        let candidates = ProjectManifest::discover(path)?;
+        let res = match candidates.last() {
             None => bail!("no projects"),
-            Some(it) => it,
+            Some(it) => it.clone(),
```

Comment:
It is now possible to have more than one project. However, we can now rely on the last project returned being the outermost one.